__tests__/app.test.js

```diff
@@ -326,4 +326,25 @@ describe("GET * (Invalid Users Routes)",()=>{
             expect(res.body.msg).toBe("Not Found");
         });
     });
+})
+describe("GET /api/articles (topic query)",()=>{
+    test("responds with status 200 and an array of articles objects filtered by the topics if given, else returns all articles",()=>{
```

Comment:
Where are your other tests? eg. empty array when no associated articles for a given topic and an error test for when the topic does not exist. Please add these.